### PR TITLE
Bump opal to 3.10.10

### PR DESCRIPTION
**Pull request by alarcher**

### CODE REVIEW

**Comment by alarcher** [comment]
Need to disable h264 or find a solution to deal with encumbered dependency.

**Comment by alarcher** [comment]
@pyhalov Should I disable x264 support so as not to depend on encumbered?

**Comment by pyhalov** [comment]
yes

**Comment by alarcher** [comment]
To be merged when #2369 is considered satisfactory.

File: components/library/opal/Makefile [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+#
+# Copyright 2014 Alexander Pyhalov.  All rights reserved.
+#
+
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME= opal
+COMPONENT_VERSION= 3.10.10
+COMPONENT_REVISION= 0
```
Comment (pyhalov) [inline components/library/opal/Makefile:20]: Please, remove COMPONENT_REVSION

**Comment by pyhalov** [comment]
Merged #2402 (with small correction).

**Pull request closed without merging**